pkg/client/didexchange/client.go

```diff
@@ -74,6 +75,7 @@ func New(ctx provider) (*Client, error) {
 		actionCh:        make(chan service.DIDCommAction, 10),
 		msgCh:           make(chan service.StateMsg, 10),
 		connectionStore: didexchange.NewConnectionRecorder(store),
+		invitationLabel: invitationLabel,
```

Comment:
I feel we don't need to save it as client variable. We can pass as an argument in [CreateInvitation](https://github.com/hyperledger/aries-framework-go/pull/638/files#diff-427522b353a28b1ec350c29c79d387b0R108), as request can have different label for invitations.

Reply:
@rolsonquadras Since we are following arcpy pattern, they use label from agent (cmd argument --label here) and client function accepts only alias name.

Reply:
@sudeshrshetty I think the question is whether we want to support dynamic labels per request or static one ?

Reply:
@rolsonquadras @sudeshrshetty  we should allow for default label and dynamic labels.